src/Project/Sugcon/SugconHomeStatic/index.html

```diff
@@ -157,8 +157,8 @@ <h5 style="text-align: left">Malaga, Spain</h5>
                                 <div class="field-promotext">
                                   <h4 style="text-align: left">ANZ</h4>
                                   <h5 style="text-align: left">SUGCON Australia and New Zealand</h5>
-                                  <h5 style="text-align: left">August 2022</h5>
-                                  <h5 style="text-align: left">Melbourne, Australia</h5>
+                                  <h5 style="text-align: left">23-24 November, 2023</h5>
```

Comment:
We're still locking down the venue for ANZ, can you change this to just read November, 2023 for now until it's confirmed?

Reply:
@robearlam , I've made the update